### ISSUE TITLE
BSTree - inorderTraversal from specific starting point

### ISSUE THREAD

**Issue by aguynamedben** [mauriciosantos/Buckets-JS#18]
Hi, this library is awesome thanks for the work you put into it.

We’re using BSTree to index domain names from documents. We can use inorderTraversal to traverse documents by our custom domain ranking.

Is there a way to inorderTraverse from a specific starting point? (vs. traversing the whole tree). It seems like some other trees provide this and maybe it could be fairly straightforward to implement in BSTree.

Use case: we want to get all docs for a certain domain (I.e. trello.com) without traversing the entire tree.

**Comment by xpusostomos** [comment]
I agree, the whole point of a btree is you can jump to somewhere and then start traversing.

**Comment by parhamsaremi** [comment]
https://github.com/mauriciosantos/Buckets-JS/pull/23
i added this feature by updating the signature to inorderTrverse(callback, element = undefined)
